src/Sidekick.Business/Apis/Poe/Models/Socket.cs

```diff
@@ -8,32 +8,32 @@ public class Socket
         public int Group { get; set; }
 
         [JsonPropertyName("sColour")]
-        public string __Color { get; set; }
+        public string ColourString { get; set; }
 
         [JsonIgnore]
-        public SocketColor Color
+        public SocketColour Colour
         {
             get
             {
-                return __Color switch
+                return ColourString switch
```

Comment:
Everywhere in our code we use `color` and `armor`. I know the json value is `colour` in this case. I'll let you judge if we keep it here only.

Reply:
I wanted to maybe try changing to Armour and Colour because the official API uses that spelling.